[2023-01-26 17:20:21,887] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-01-26 17:20:21,902] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-01-26 17:20:21,902] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 17:20:21,902] {taskinstance.py:1377} INFO - Starting attempt 8 of 10
[2023-01-26 17:20:21,902] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 17:20:21,918] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Leer_Datos_Respuesta_Nasa> on 2023-01-02 00:00:00+00:00
[2023-01-26 17:20:21,923] {standard_task_runner.py:52} INFO - Started process 1081 to run task
[2023-01-26 17:20:21,927] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Leer_Datos_Respuesta_Nasa', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '601', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpf0tjr67s', '--error-file', '/tmp/tmph4z5cw96']
[2023-01-26 17:20:21,929] {standard_task_runner.py:80} INFO - Job 601: Subtask Leer_Datos_Respuesta_Nasa
[2023-01-26 17:20:22,000] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-02T00:00:00+00:00 [running]> on host afaa1e0ff064
[2023-01-26 17:20:22,082] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=davidmontenegro2403@gmail.com
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Leer_Datos_Respuesta_Nasa
AIRFLOW_CTX_EXECUTION_DATE=2023-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=8
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-02T00:00:00+00:00
[2023-01-26 17:20:22,083] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-26 17:20:22,084] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /tmp && head /tmp/response_20230102.txt']
[2023-01-26 17:20:22,096] {subprocess.py:85} INFO - Output:
[2023-01-26 17:20:22,099] {subprocess.py:92} INFO - ***tmpbrtmnwu9
[2023-01-26 17:20:22,100] {subprocess.py:92} INFO - pymp-839nzyvl
[2023-01-26 17:20:22,100] {subprocess.py:92} INFO - response_20230101.txt
[2023-01-26 17:20:22,100] {subprocess.py:92} INFO - response_20230102.txt
[2023-01-26 17:20:22,100] {subprocess.py:92} INFO - response_20230103.txt
[2023-01-26 17:20:22,100] {subprocess.py:92} INFO - tmpac_2qo7u
[2023-01-26 17:20:22,100] {subprocess.py:92} INFO - tmph4z5cw96
[2023-01-26 17:20:22,100] {subprocess.py:92} INFO - tmpmlqsfa3t
[2023-01-26 17:20:22,100] {subprocess.py:92} INFO - tmpnvc2r2ky
[2023-01-26 17:20:22,101] {subprocess.py:92} INFO - Confirmaci√≥n de la NASA, pueden proceder
[2023-01-26 17:20:22,101] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-26 17:20:22,135] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Leer_Datos_Respuesta_Nasa, execution_date=20230102T000000, start_date=20230126T172021, end_date=20230126T172022
[2023-01-26 17:20:22,179] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 17:20:22,228] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
